meta.go

```diff
@@ -239,13 +239,16 @@ func (m *MetaSpec) cleanParameters(metaInterface map[string]interface{}) (map[st
 	copyParamValuesIntoMap(ret, parameters)
 
 	// Override the values with job-specific ones for this jobName
-	jobName, _ := m.Get("build.jobName")
-	jobRE := parentJobNameRegExp.FindStringSubmatch(jobName)
-	jobName = jobRE[2]
-	if _, jobParameters := fetchMetaValue(jobName, parameters); jobParameters != nil {
-		copyParamValuesIntoMap(ret, jobParameters)
-	} else {
-		logrus.Tracef("No jobParameters for jobName: %s", jobName)
+	_, buildJobName := fetchMetaValue("build.jobName", metaInterface)
```

Comment:
Should we remove the receiver argument `(m *MetaSpec)` from the function signature?

Reply:
oh, nice - I guess the .Get was the only reason it was there… sure

Reply:
Done